src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -309,7 +300,6 @@ public function load(array $configs, ContainerBuilder $container)
                 'Symfony\\Component\\DependencyInjection\\Container',
 
                 'Symfony\\Component\\EventDispatcher\\Event',
-                'Symfony\\Component\\EventDispatcher\\ContainerAwareEventDispatcher',
```

Comment:
without this change it would add to class cache non-existent class. Maybe better #22786 first